### ISSUE TITLE
Autopilot Installation

### ISSUE THREAD

**Issue by CMDRZOD** [ElementsProject/lightning#3625]
### Issue and Steps to Reproduce

Just trying to get the autopilot plugin installed. I've installed the dependencies but I keep getting the following output, and the node closes process.

```
2020-04-06T20:51:35.607Z DEBUG plugin-manager: started(18529) /usr/local/bin/../libexec/c-lightning/plugins/fundchannel
2020-04-06T20:51:35.608Z DEBUG plugin-manager: started(18530) /usr/local/bin/../libexec/c-lightning/plugins/pay
2020-04-06T20:51:35.609Z DEBUG plugin-manager: started(18531) /usr/local/bin/../libexec/c-lightning/plugins/bcli
2020-04-06T20:51:35.609Z DEBUG plugin-manager: started(18532) /usr/local/bin/../libexec/c-lightning/plugins/autoclean
2020-04-06T20:51:35.611Z DEBUG plugin-manager: started(18533) /home/hair/plugins/autopilot/autopilot.py
2020-04-06T20:51:35.793Z INFO plugin-autopilot.py: RPC method 'autopilot-run-once' does not have a docstring.
lightningd: no arguments accepted
```

### `getinfo` output
```json
{
   "id": "02a20247f515d978cbf9e9ce6a4287b5931d724068b7b88bbaead6380db3dd8e9a",
   "alias": "Nasty_Nardo",
   "color": "095821",
   "num_peers": 10,
   "num_pending_channels": 0,
   "num_active_channels": 10,
   "num_inactive_channels": 0,
   "address": [
      {
         "type": "torv2",
         "address": "v2kdcetvslmdfcwr.onion",
         "port": 9735
      }
   ],
   "binding": [
      {
         "type": "ipv4",
         "address": "127.0.0.1",
         "port": 9735
      }                                                                                                                                                                ],                                                                                                                                                                  "version": "v0.8.1-180-gca36c21",                                                                                                                                   "blockheight": 624723,                                                                                                                                              "network": "bitcoin",                                                                                                                                               "msatoshi_fees_collected": 0,
   "fees_collected_msat": "0msat",
   "lightning-dir": "/home/hair/.lightning/bitcoin"
}
```

**Comment by ZmnSCPxj** [comment]
The autopilot program needs to be updated to run on latest version, or you need to downgrade your C-Lightning installation (which is probably not safe to do, we do not expect C-Lightning version downgrades; you would have to destroy your node and recreate it in the older C-Lightning version).

**Comment by cdecker** [comment]
The error with which `lightnignd` exits seems unrelated to the autopilot plugin (which produces just some warnings about missing docstrings in the plugin, but doesn't cause the termination). The following lines seem to produce the error causing the exit:

https://github.com/ElementsProject/lightning/blob/2f1502abf4ed356273c244d7d9116cc05f88c288/lightningd/options.c#L1095-L1098

Could you share the command line and the config that you use to start the c-lightning node?

**Comment by CMDRZOD** [comment]
I've actually found a copy paste error in my command line. C-lightning now starts without issue. Terribly sorry to have started a issue for this, looks like its time for me to make an easier to edit config. 
You'll see my error below:

`lightningd --network=bitcoin --log-level=debug --bitcoin-rpcuser=rpcuser --bitcoin-rpcpassword=rpcpass --alias=(redacted) --rgb=095821 lightningd --plugin=/home/hair/plugins/autopilot/autopilot.py --autopilot-percent 80 --autopilot-num-channels 25 --autopilot-min-channel-size-msat 300000000 --log-level=debug --proxy=127.0.0.1:9050 --bind-addr=127.0.0.1:9735 --announce-addr=redacted.onion`

@ZmnSCPxj Now that I've got autopilot to start with my node, I think I will try and experiment and see what happens. Should I open any issues I encounter or is the dev team currently focused on other parts of clightning?
